.yamato/universal-android-opengles3.yml

```diff
@@ -49,7 +49,7 @@ Build_Universal_Android_OpenGLES3_Player_trunk:
       -  mklink /d WindowsEditor\Data\PlaybackEngines\AndroidPlayer\OpenJDK %JAVA_HOME% || exit 0
       -  mklink /d WindowsEditor\Data\PlaybackEngines\AndroidPlayer\SDK %ANDROID_SDK_ROOT% || exit 0
       -  mklink /d WindowsEditor\Data\PlaybackEngines\AndroidPlayer\NDK %ANDROID_NDK_ROOT% || exit 0
-      -  utr --suite=playmode --platform=Android --testproject=TestProjects\UniversalGraphicsTest --editor-location=WindowsEditor --artifacts_path=test-results --player-save-path=players --scripting-backend=il2cpp --timeout=1800 --build-only
+      -  utr --suite=playmode --platform=Android --testproject=TestProjects\UniversalGraphicsTest --editor-location=WindowsEditor --artifacts_path=test-results --timeout=1200 --player-save-path=players --build-only --scripting-backend=il2cpp
```

Comment:
It seems timeout=1200 has slipped through for this build step (was at 1800 before)

Reply:
(Same for fast-trunk and CUSTOM-REVISION)

Reply:
(And for the other APIs 😄)

Reply:
Good catch! Double-checking all of these now